cyral/resource_cyral_repository.go

```diff
@@ -429,15 +440,31 @@ func resourceRepository() *schema.Resource {
 							Optional:    true,
```

Comment:
Here, the ideal behavior should be "if the server type is replicaset, then this is actually mandatory". If we can express that in code, that would be nice.

Regardless, the default here should just be an empty string.